client/api/src/execution_extensions.rs

```diff
@@ -126,8 +126,10 @@ impl<Block: traits::Block> ExecutionExtensions<Block> {
 	/// extension to be a `Weak` reference.
```

Comment:
This comment needs to be updated although we're still doing it internally.

Reply:
Yeah, I'll just make that an internal comment and not a doc one.